CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-## v8.1.2 (Backport of v12.0.1)
+## v8.1.2
+
+**This release backports specific changes from v12.0.1 and v14.0.0**
```

Comment:
`v8.1.2` has already been released with changes from `v12.0.1`. I think that this should be `v8.1.3`